src/libs/protocols/hack/update-generated-proto.sh

```diff
@@ -81,8 +83,8 @@ fi;
 which protoc
 [ $? -eq 0 ] || die "Please install protoc from github.com/protocolbuffers/protobuf"
 
-which protoc-gen-gogottrpc
-[ $? -eq 0 ] || die "Please install protoc-gen-gogottrpc from https://github.com/containerd/ttrpc"
+which protoc-gen-gottrpc
+[ $? -eq 0 ] || die "Please install protoc-gen-gottrpc from https://github.com/containerd/ttrpc"
```

Comment:
Hi @wainersm maybe is outdated, yes. They still have binaries for code generation for both... https://github.com/containerd/ttrpc/tree/main/cmd.

I choose to keep with containerd/ttrpc, because of a few things:

1. We depend on containerd, then aligning this project to use ttrpc would ensure compatibility;
2. Another alternative would be to adopt [protobiuld](https://github.com/containerd/protobuild). But I would like to make this change as simple as possible, since it is already a big change.
3. If future versions of containerd move away from ttrpc, then we might want to re-evaluate whether to continue using ttrpc or not.